src/main/java/io/github/opencubicchunks/cubicchunks/CubicChunksConfig.java

```diff
@@ -0,0 +1,30 @@
+package io.github.opencubicchunks.cubicchunks;
+
+import com.electronwill.nightconfig.core.file.CommentedFileConfig;
+import com.electronwill.nightconfig.core.io.WritingMode;
+import net.minecraftforge.common.ForgeConfigSpec;
+import net.minecraftforge.fml.common.Mod;
+
+import java.io.File;
+
+@Mod.EventBusSubscriber
+public class CubicChunksConfig {
+    private static final ForgeConfigSpec.Builder builder = new ForgeConfigSpec.Builder();
+    public static final ForgeConfigSpec config;
+
+    public static ForgeConfigSpec.IntValue verticalViewDistance;
+
+    static {
+        verticalViewDistance = builder
+                .comment("Vertical radius of cubes to load")
+                .defineInRange("verticalViewDistance", 4, 2, 16);
```

Comment:
Match vanilla view distance range
```suggestion
                .defineInRange("verticalViewDistance", 4, 2, 32);
```